### ISSUE TITLE
Port to new project file

### ISSUE THREAD

**Issue by dreammelter** [dreammelter/NVD23#1]
Move the working bits to a clean project file and record the steps for the handout.

- [x] Project and Engine Settings
- [x] Art assets, Project Structure
- [x] basic Player setup
- [x] basic (sandbox) level setup
 ❌ basic brick breaking mechanics (Affected by #27 so we're skipping this part.)

**Comment by dreammelter** [comment]
The base mechanics for the block breaking stuff should include the Block spawner, but I'll leave that in a separate issue for now: #6

**Comment by dreammelter** [comment]
Wall jump sprite is slightly off for the pixel adventure pink guy (and possibly the rest)... if we want to perfect it (to implement #24 cleanly) we could just flip the sprite sheet.

Doesn't have to be perfect for this step though.

**Comment by dreammelter** [comment]
...wondering if we'll really need to introduce a state machine for the player character. The code is pretty simple + clean as it is.

**Comment by dreammelter** [comment]
I promise I'll switch to a new branch once this issue's done.

**Comment by dreammelter** [comment]
🚨 Not sure what the difference between the sandbox where I prototyped the paddle behavior and this new file, but things don't work like they did. (Which figures! When does anything work when you take it out of its perfect black box of conditions.... but that's the exact reason I started a new file. It's gotta work everywhere.)

Gonna ditch the paddle for now. I've managed to minimize the amount of skewing following a path did to the sprite, but I'd need more testing time to make it playable again. Which means there's a design change that's gonna push this out again... the paddle did seem like it'd be obnoxious in the long run.
